src/pigui/ModelSpinner.cpp

```diff
@@ -118,7 +118,7 @@ void ModelSpinner::DrawPiGui()
 
 	const ImGuiIO &io = ImGui::GetIO();
 	bool hovered = ImGui::IsItemHovered();
-	if (hovered && ImGui::IsMouseDown(0)) {
+	if (hovered && ImGui::IsMouseDown(SDL_BUTTON_MIDDLE)) {
```

Comment:
SDL and ImGui have different mouse button numbering orders, AFAIK, i.e. even if it works in this case, it is wrong in meaning.

Reply:
OK. Got it!